package.json

```diff
@@ -164,7 +163,7 @@
 		"whatwg-fetch": "1.1.1"
 	},
 	"devDependencies": {
-		"@automattic/calypso-build": "1.0.0-alpha.0",
+		"@automattic/calypso-build": "1.0.0-alpha.2",
```

Comment:
Already done in https://github.com/Automattic/jetpack/pull/11941

Reply:
Fixed by a rebase.